app/components/widgets/inputs/ImageLayout.vue

```diff
@@ -0,0 +1,5 @@
+<template>
+  <image-picker :value="value" :metadata="meta" @input="value => emitInput(value)" />
+</template>
+
+<script lang="ts" src="./ImageLayout.vue.ts"></script>
```

Comment:
We use `Input` postfix for the classes that we extend from BaseInput. I think it is a good convention to follow. I have the same comment for the  `image-picker`